src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,99 @@
+const expensesService = require('../services/expenses.service');
+const usersService = require('../services/users.service');
+
+const getAll = async (req, res) => {
+  const { from, to, userId, categories } = req.query;
+
+  res.send(
+    expensesService.getAll({
+      from,
+      to,
+      userId,
+      categories,
+    }),
+  );
+};
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const { userId, title, amount, category, note, spentAt } = req.body;
+
+  if (!userId || !title || !amount || !category || !spentAt) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (!usersService.getById(userId)) {
```

Comment:
Can you move this check inside previous `if`?
You return the same result in both of them